### DISCUSSION TITLE
手机可以连上VPS但是电脑连不上

### DISCUSSION THREAD

**Discussion by JasonL111** [v2fly/v2ray-core#1815] [Q&A]
使用的是板瓦工，v2ray已经配置完成（手机能连上并浏览外网）。我使用v2rayN并且按照教程进行了客户端搭建，没有成功。于是更换v2rayW，再次失败。目前已排除服务器防火墙影响的可能，已经排除服务器配置文件错误的可能，节点有问题的可能，浏览器插件导致连不上的可能。客户端的报告已经上传（ip及端口已更改）

**Comment by JasonL111** [comment]
已解决，是浏览器代理的问题

**Reply by maliang1234m** [reply]
我重新卸载了，然后又来了次，这次没用订阅。。。好像一切正常，过两天看看，浏览器代理是改什么，在哪里啊？
